openid-4-verifiable-presentations-1_0.md

```diff
@@ -357,23 +357,52 @@ Content-Type: application/json
 
 ## Using `scope` Parameter to Request Verifiable Credential(s) {#request_scope}
 
-Wallets MAY support requesting presentation of Verifiable Credentials using OAuth 2.0 scope values.
+Wallets MAY support requesting presentation of Verifiable Credentials using OAuth 2.0 `scope` values.
 
-Such a scope value MUST be an alias for a well-defined Presentation Definition that will be 
-referred to in the `presentation_submission` response parameter. 
+In the case where all the Input Descriptors defined within the Presentation Definition are required within the request, the value contained in the scope parameter MUST match to the Presentation Definition id value. In the case where only a subset of Verifiable Credentials is required, each value contained in the scope parameter MUST match to one Input Descriptor(s) `id` value, as defined within the Presentation Definition.
 
-The specific scope values, and the mapping between a certain scope value and the respective 
-Presentation Definition is out of scope of this specification. 
+Since the values contained in the `scope` parameter are expressed as a list of space-delimited and case-sensitive strings, according to [@RFC6749], the Verifier that supports the `scope` parameter to request Verifiable Credential(s) MUST NOT include spaces in the Input Descriptors `id` values intended to be used within the `scope` parameter. It is then RECOMMENDED to use collision-resistant scopes values.
 
-Possible options include normative text in a separate specification defining scope values along with a description of their
-semantics or machine readable definitions in the Wallet's server metadata, mapping a scope value to an equivalent 
-Presentation Definition JSON object. 
+It also required that the Presentation Definition id value MUST be different to the ones defined for the Input Descriptors defined within it.
 
-Such definition of a scope value MUST allow the Verifier to determine the identifiers of the Presentation Definition and Input Descriptor(s) in the `presentation_submission` response parameter (`definition_id` and `descriptor_map.id` respectively) as well as the Credential formats and types in the `vp_token` response parameter defined in (#response-parameters).  
+The Verifier that intends to use this feature MUST publish its Presentation Definition within its metadata, using the parameter name `presentation_definition`.
```

Comment:
why MUST? I think it is possible that wallet and the verifier agree on the scope values out of band?

Reply:
I agree

Reply:
Done here baf2dd46ac16a0337c653040a27aeb8efde0c331